app.cr

```diff
@@ -27,6 +43,22 @@ def filter(repos, filter)
   filtered
 end
 
+def sort(repos, sort)
+  sorted = repos.dup
+  if sort == "stars"
```

Comment:
I think it is a bit better here is to use `case` statement instead of multiple `if`s

```crystal
case sort
when "stars"
  #
when "updated"
  #
when "forks"
  #
end
```